server/executor/trigger/http.go

```diff
@@ -10,23 +10,17 @@ import (
 	"strings"
 
 	"github.com/kubeshop/tracetest/server/model"
-	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
-	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/propagation"
 	"go.opentelemetry.io/otel/trace"
 )
 
 func HTTP() Triggerer {
-	return &httpTriggerer{
-		traceProvider: traceProvider(),
-	}
-}
-
-type httpTriggerer struct {
-	traceProvider *sdktrace.TracerProvider
+	return &httpTriggerer{}
 }
 
-func (te *httpTriggerer) Trigger(_ context.Context, test model.Test, tid trace.TraceID, sid trace.SpanID) (Response, error) {
+type httpTriggerer struct{}
 
+func (te *httpTriggerer) Trigger(_ context.Context, ctx context.Context, test model.Test, tid trace.TraceID, sid trace.SpanID) (Response, error) {
```

Comment:
this looks like the first context should be called `ctx` and the 2nd `triggerSpanCtx` or something like that